ext/date/tests/examine_diff.inc

```diff
@@ -47,9 +47,10 @@ function examine_diff($end_date, $start_date, $expect_spec, $expect_days, $absol
     }
     $end_date = $end->format('Y-m-d H:i:s T');
 
+    $invert = false;
```

Comment:
What was the reason to change this? Adding and subtracting aren't necessarily idempotently opposite.

Reply:
I had to make this change because the `$invert` property became `readonly`. But if it's a legitimate use-case to modify it, then I can revert the `readonly` flag.